servers/src/main/java/tachyon/master/file/FileSystemMaster.java

```diff
@@ -1795,6 +1819,141 @@ private void setStateFromEntry(SetStateEntry entry) throws FileDoesNotExistExcep
   }
 
   /**
+   * Sets the acl of a file or directory. Only one of owner, group, or permission in the
```

Comment:
in order to be consistent with client side, where we cannot change more than one at a time. Say there is no command to change owner and group together.